iwyu_path_util.cc

```diff
@@ -178,9 +177,13 @@ string GetParentPath(const string& path) {
 
 // Converts a file-path, such as /usr/include/stdio.h, to a
 // quoted include, such as <stdio.h>.
-string ConvertToQuotedInclude(const string& filepath) {
+string ConvertToQuotedInclude(const string& filepath, const string& includerPath) {
```

Comment:
That does seem to be the prevailing style, I hadn't really noticed until I felt `includePath` looked weird :-)

I'm sure there are exceptions, but the vast majority seems to be all_lower_with_underscore.